healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceGetHistory.java

```diff
@@ -37,15 +37,15 @@
 import org.apache.http.impl.client.HttpClients;
 
 public class FhirResourceGetHistory {
-  private static final String FHIR_NAME = "projects/%s/locations/%s/datasets/%s/fhirStores/%s";
+  private static final String FHIR_NAME = "projects/%s/locations/%s/datasets/%s/fhirStores/%s/fhir/%s/%s";
   private static final JsonFactory JSON_FACTORY = new JacksonFactory();
   private static final NetHttpTransport HTTP_TRANSPORT = new NetHttpTransport();
 
   public static void fhirResourceGetHistory(String resourceName, String versionId)
       throws IOException, URISyntaxException {
     // String resourceName =
     //    String.format(
-    //        FHIR_NAME, "project-id", "region-id", "dataset-id", "store-id", "fhir-id");
+    //        FHIR_NAME, "project-id", "region-id", "dataset-id", "store-id", "fhir-id", "resource-type", "resource-id");
```

Comment:
There are 6 `%s` in `FHIR_NAME` and 7 params here.  Something is amis.

Reply:
Ah looks like "store-id" and "fhir-id" were meant to be "FHIR store ID" and "FHIR Resource ID". You can delete "fhir-id" or "store-id" or combine them into "fhir-store-id".

Reply:
Whoops, thanks. Deleted "fhir-id".